redaxo/src/core/lib/Fragment/Component/Alert.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
in redaxo haben wir sonst immer nur klassen mit lower-case namen. mir sind die dateinamen gleich ins auge gefallen weil wir hier jetzt mit upper-case first dateien/ordner namen arbeiten (kein nogo)